docs/auth/jwt/jwt-configuration.mdx

```diff
@@ -21,24 +21,29 @@ This section describes the JSON Web Token (JWT) configuration options available
 
 Example JSON Web Token (JWT) payload configuration definition:
 
-```json
-{
-  "type": "optional-type-of-key",
-  "key": "optional-key-as-string",
-  "jwk_url": "optional-url-to-refresh-jwks",
-  "sub": "optional-string-to-verify-subject",
-  "claims_namespace": "optional-key-name-in-claims",
-  "claims_namespace_path": "optional-json-path-to-the-claims",
-  "claims_format": "json|stringified_json",
-  "audience": "optional-string-or-list-of-strings-to-verify-audience",
-  "issuer": "optional-string-to-verify-issuer",
-  "claims_map": "optional-object-of-session-variable-to-claim-jsonpath-or-literal-value",
-  "allowed_skew": "optional-number-of-seconds-in-integer",
-  "header": "optional-key-to-indicate-cookie-or-authorization-header"
-}
+```yaml title="globals/metadata/auth-config.hml"
```

Comment:
@hgiasac So this section has changed to be about the AuthConfig definition rather than the JWT token definition. Do we not need a JWT payload definition also?

Reply:
No. That JWT definition is a [JSON string](https://hasura.io/docs/2.0/auth/authentication/jwt/#configure-hasura-jwt-mode) in v2 when configuring the `HASURA_GRAPHQL_JWT_SECRET` env variable. In Hasura DDN, JWT is just an auth mode of `AuthConfig` and the syntax is different from v2.

We can link the definition to the [AuthConfig reference](https://hasura.io/docs/3.0/reference/metadata-reference/auth-config#jwt) instead.